src/move/move.spec.ts

```diff
@@ -159,3 +156,23 @@ describe("checkUserMove", () => {
     expect(allowedMove).toBeFalsy();
   });
 });
+
+describe("getPlayableTilesForNextMove", () => {
+  it("shouldn't have any possibility for lone tile", () => {
+    const playableTiles = getPlayableTilesForNextMove(mockBoard, {
+      symbol: "O",
+      color: "grey",
+    });
+
+    expect(playableTiles.length).toEqual(0);
+  });
+
+  it("should have 6 possible moves for red D tile", () => {
+    const playableTiles = getPlayableTilesForNextMove(mockBoard, {
+      symbol: "D",
+      color: "red",
+    });
+
+    expect(playableTiles.length).toEqual(6);
```

Comment:
would have been even better to actually check that the `playableTiles` are the ones you expect, but I guess it's good enough given the time we have left :sweat_smile:

Reply:
Done !